### DISCUSSION TITLE
Budibase CLI BackUp SSL Causing Issue

### DISCUSSION THREAD

**Discussion by JayP718** [Budibase/budibase#10009] [Help]
Hi all!

I am trying to back up my budibase instance. I am using a self signed certificate which is working great for the front end application. Unfortunately anytime I try to update I receive the following error.

It seems like the CouchDB portion works fine after setting NODE_TLS_REJECT_UNAUTHORIZED to 0.

However, it still get's hung up on the MINIO portion.

budi backups --export --env .env2
(node:21836) Warning: Setting the NODE_TLS_REJECT_UNAUTHORIZED environment variable to '0' makes TLS connections and HTTPS requests insecure by disabling certificate verification.
(Use `node --trace-warnings ...` to show where the warning was created)
CouchDB Export
 ████████████████████████████████████████ 100% | ETA: 0s | 20/20
S3 Export
Error - Error: Unable to access MinIO/S3 - check environment config.
root@115bcd6dc7f6:/# Connection to nysap closed by remote host.
Connection to nysap closed.

**Comment by JayP718** [comment]
Any help is appreciated, thanks!

**Reply by melohagan** [reply]
Hey @JayP718 

I'm wondering why you need to use SSL at all?
Ideally you should run the Budibase backup commands local to the location that your Budibase services are running. 

Would you be able to provide the contents of your environment file as well please?

**Reply by JayP718** [reply]
Hi Mel,

So I had to make some changes to the Nginx proxy to allow for SSO. Because we are forced to use https url for the redirect. Perhaps there's some changes I can make to nginx.conf to also allow http auth?
